lib/Plugin.php

```diff
@@ -230,11 +230,21 @@ private function define_commands() : void {
 		} );
 	}
 
+	/**
+	 * Define the integrations with other plugins.
+	 *
+	 * @since Unreleased Register ACF integration immediatly for field registration.
+	 * @since 0.0.1
+	 *
+	 * @return void
+	 */
 	private function define_integrations() : void {
 		$this->define_integration_migrators();
+		$immediate_integrations = [
```

Comment:
Don't know if it's an overkill for now, but it seems that we might need other situations for this and we could structure things a bit better in an array that indicates when to run each integration.